control-services/cli/gitea.py

```diff
@@ -119,16 +119,18 @@ def __createConfigUser(self, password=None):
             ' --must-change-password=false'
         )
 
-    def __copyLatestContentRepo(self):
+    def __copyLatestContentRepo(self, branch):
         # Pull from remote to local, assuming local does not have uncommitted changes
         # git --git-dir /home/control/$CONFIG_REPO_NAME/.git pull
-        out = check_output(
-            [
-                'git', '--git-dir',
-                self.cfg['host']['content_git_dir_path'], 'pull'
-            ],
-            universal_newlines=True
-        )
+        
+        #out = check_output(
```

Comment:
This was removed because `sync` should only ensure that `control-rous` and `gitea-rous` are the same.

Reply:
If this is no longer needed, we should delete it rather than commenting it out